src/utils/CMakeLists.txt

```diff
@@ -56,7 +56,7 @@ endforeach(i)
 
 # some regular UTILS tools need the GUI lib, only build them when WITH_GUI is enabled
 if(WITH_GUI)
-  find_package(Qt5 COMPONENTS Core Network Widgets Svg OpenGL REQUIRED)
+  find_package(Qt5 COMPONENTS Core Network Widgets WebEngineWidgets Svg OpenGL REQUIRED)
```

Comment:
TODO make WebEngineWidgets optional

Reply:
In order to make that package optional, I need to move it to after the REQUIRED keyword right? 
find_package(Qt5 COMPONENTS Core Network Widgets Svg OpenGL REQUIRED WebEngineWidgets)